src/config/env.ts

```diff
@@ -33,6 +35,8 @@ export const REACT_APP_HASURA_URL = getEnvValue(
   'https://missing-hasura-url.edu'
 );
 
+export const LOCAL_HASURA_PORT = process.env.LOCAL_HASURA_PORT;
```

Comment:
i realize now that using LOCAL_ and CI_ prefixes is an antipattern... will have to go back and fix this at some point. just FYI, not a blocker